### PR TITLE
wiki + java-tron + contributions + contact merged

### PR DESCRIPTION
**Pull request by styk-tv**
All materials from wiki and java-tron repos in English versions are now available in wiki.tron.network  Preview is here:  http://wiki.tron.network/en/future/introduction.html  once you approve it, this will become the new prod wiki on main address. All images used by wiki are inside of /docs/img folder we can slowly re-map everything from outside to be referenced from wiki as opposed to repo. I will then be able to remove /images/ folder. For now we're up to date with English. Should start adding CN in next tasks.

### CODE REVIEW

**Comment by ruchern** [comment]
I’ll see to it in the morning when I wake up.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[![Documentation Status](https://readthedocs.org/projects/tron-wiki/badge/?version=latest)](http://tron-wiki.readthedocs.io/en/latest/?badge=latest)
+                
+# TRON Wiki
+
+## Getting Started
+The documentation is currently hosted on http://wiki.tron.network
+
+## Contributing
```
Comment (styk-tv) [inline README.md]: @ruchern can we please remove those two empty sections for now so we can publish this. you can then work on those at later time.

**Comment by styk-tv** [comment]
@tronfoundation @lucien-tronlab @sasaxie @ruchern can you please send me Slack invite to my github published address? cheers.

**Comment by sasaxie** [comment]
Look good to me.

File: README.md (hunk 1 above)
Comment (ruchern) [reply README.md]: Sorry, I will take this out now.

**Comment by ruchern** [comment]
@styk-tv Is there a Slack channel? I am aware of that.

**Comment by ruchern** [comment]
Oh, it seemed like you need a `@tronlab.com` email address to access that. I hope to get an invitation too.

@sasaxie

**Comment by styk-tv** [comment]
@ruchern self-sign on tronlab everyone else invite

**Comment by xiechang33** [comment]
@styk-tv @ruchern Please send me your email address, I will send you the Slack invite.Here is my email:xiechang@peiwo.cn

**Comment by ybhgenius** [comment]
Hello, I am a crew member of TRON, and now we are planning to provide an incentive for your contribution. Please leave me your email address. Thanks.

**Pull request merged**